casper/src/test/scala/coop/rchain/casper/util/rholang/RuntimeManagerTest.scala

```diff
@@ -71,7 +69,7 @@ class RuntimeManagerTest extends FlatSpec with Matchers {
     result.status.isFailed should be(true)
   }
 
-  "computeDeployPayment" should "transfer REV from the deployer's vault to the PoS vault" ignore withTestFixture {
+  "computeDeployPayment" should "transfer REV from the deployer's vault to the PoS vault" in withTestFixture {
```

Comment:
Same here. Should be done in a fixup.

Reply:
there are conflicts and the fix-up it only works if I put everything in one big commit but if you insist...